app/views/layouts/application.html.slim

```diff
@@ -21,14 +21,12 @@ html.govuk-template.app-html-class lang="en" class=("modal" if current_variant?(
     = csrf_meta_tags
 
   body class=body_class data-vacancy_state=(@vacancy.present? ? @vacancy.state : "create")
-    - if current_variant?(:"2021_04_cookie_consent_test", :modal) && !cookies_preference_set?
-      .modal-container
     = render partial: "shared/google_tag_manager_body" if cookies["consented-to-cookies"] == "yes"
     = javascript_tag(nonce: true) do
       | document.body.className = ((document.body.className) ? document.body.className + " js-enabled" : "js-enabled");
     = render "shared/skip_links"
-    - unless cookies_preference_set? || %i[top none].include?(ab_variant_for(:"2021_04_cookie_consent_test"))
-      = render(AbTest::CookiesBannerComponent.new(create_path: create_cookies_preferences_path(cookies_consent: "yes", **utm_parameters), reject_path: create_cookies_preferences_path(cookies_consent: "no", **utm_parameters), preferences_path: cookies_preferences_path(**utm_parameters)).with_variant(ab_variant_for(:"2021_04_cookie_consent_test")))
+    - unless cookies_preference_set? || %i[bottom_sticky_default bottom_sticky_gds].include?(ab_variant_for(:"2021_05_cookie_consent_test"))
```

Comment:
Sometimes I find this way round is easier to read in certain contexts (although I can never decide a clear cut, nicely defined rule for when to use in/include)
```ruby
ab_variant_for(:"2021_05_cookie_consent_test").in?(%i[bottom_sticky_default bottom_sticky_gds])
```